tests/multiprocessing_smoke_tests.py

```diff
@@ -12,62 +13,65 @@
                                 AnnealingBoxCurriculum)
 from syllabus.tests import (create_cartpole_env,
                             create_nethack_env,
-                            get_test_values,
-                            test_native_multiprocess,
-                            test_ray_multiprocess,
-                            test_single_process)
+                            get_test_values)
+
+from syllabus.tests import test_single_process as run_single_process
+from syllabus.tests import test_native_multiprocess as run_native_multiprocess
+from syllabus.tests import test_ray_multiprocess as run_ray_multiprocess
 
 N_ENVS = 2
 N_EPISODES = 2
 
-if __name__ == "__main__":
-    ray.init()
-    nethack_env = create_nethack_env()
-    cartpole_env = create_cartpole_env()
-    curricula = [
+
+ray.init()
+nethack_env = create_nethack_env()
+cartpole_env = create_cartpole_env()
+
+curricula = [
         (NoopCurriculum, create_nethack_env, (NetHackScore, nethack_env.task_space), {}),
         (DomainRandomization, create_nethack_env, (nethack_env.task_space,), {}),
         # (LearningProgressCurriculum, create_nethack_env, (nethack_env.task_space,), {}),
         (CentralizedPrioritizedLevelReplay, create_nethack_env, (nethack_env.task_space,), {"device": "cpu", "suppress_usage_warnings": True, "num_processes": N_ENVS}),
-        (PrioritizedLevelReplay, create_nethack_env, (nethack_env.task_space, nethack_env.observation_space), {
-            "get_value": get_test_values,
-            "device": "cpu",
-            "num_processes": N_ENVS,
-            "num_steps": 2048
-        }),
+        #(PrioritizedLevelReplay, create_nethack_env, (nethack_env.task_space, nethack_env.observation_space), {
+        #    "get_value": get_test_values,
+        #    "device": "cpu",
+        #    "num_processes": N_ENVS,
+        #    "num_steps": 2048
+        #}),
```

Comment:
What happened here? Why is this commented out?

Reply:
As per our discussion on `support` channel, it was recommended to comment out those tests which doesn't work. `LearningProgressCurriculum` fails to pass current setup (as algorithm has been updated and it breaks the code) and `PrioritizedLevelReplay` test hangs for `multiprocessing_queue `tests for a long time. Although `PrioritizedLevelReplay` passes for` single process` and `ray multiprocess`. That's why these two are commented out.